pyscriptjs/src/pyconfig.ts

```diff
@@ -145,17 +145,18 @@ function mergeConfig(inlineConfig: AppConfig, externalConfig: AppConfig): AppCon
 
 function parseConfig(configText: string, configType = 'toml') {
     if (configType === 'toml') {
+        // TOML parser is soft and can parse even JSON strings, this additional check prevents it.
```

Comment:
do we still need this additional check @hoodmane or the new library fixes this?

Reply:
I think it's the same library and the same checks are needed. We can try removing it in a followup but I would prefer to merge this as-is.